deploy/overlays/helm/delete-ns.yaml

```diff
@@ -0,0 +1,5 @@
+$patch: delete
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: security-profiles-operator
```

Comment:
Let's add a newline at the end of the file otherwise editors will add whitespace

Reply:
sure